### PR TITLE
[HPO] fix bug occurred when running HPO without batch size in search space

### PR DESCRIPTION
**Pull request by eunwoosh**
This PR fixes a bug which is occurred when HPO runs without batch size in search space.

### CODE REVIEW

**Comment by eunwoosh** [comment]
run ote-test

File: ote_cli/ote_cli/utils/hpo.py [hunk 1]
```diff
@@ -232,6 +232,13 @@ def run_hpo_trainer(
         else:
             hyper_parameters.learning_parameters.num_iters = hp_config["iterations"]
 
+    # add batch size value if batch size isn't in search space
```
Comment (yunchu) [inline ote_cli/ote_cli/utils/hpo.py]: this comment should be changed like
"current HPO requires batch_size information so if batch_size is not in the HPO's target hparams, need to set batch_size config from the instance of ConfigurableParameters"

Comment (eunwoosh) [reply ote_cli/ote_cli/utils/hpo.py]: I see. I modified as you said.

**Review by goodsong81 — APPROVED** [review]

**Review by supersoob — APPROVED** [review]
LGTM

**Review by yunchu — APPROVED** [review]

**Pull request merged**